docs/interfaces.md

```diff
@@ -107,6 +107,17 @@ Auto-Connect: no
 
 ## Supported Interfaces - Advanced
 
+### browser
```

Comment:
We another name for the interface, as saying plugs: [browser] should imply _talking_ to a browser rather than _being_ a browser, per existing conventions.